msvc/libmonoruntime.vcxproj

```diff
@@ -28,6 +28,9 @@
   <ItemGroup>
     <None Include="..\mono\metadata\Makefile.am" />
   </ItemGroup>
+  <ItemGroup>
+    <ClCompile Include="..\mono\metadata\oop.c" />
+  </ItemGroup>

```

Comment:
This file should be added to the libmonoruntime-common.targets file instead of the msvc/libmonoruntime.vcxproj file. Currently, we're separating the list of files to compile from the project which indicates how to compile them. That might change in the future, but let's stick to it for now.